backend/v3/storage/database/events_testing/id_provider_instance_test.go

```diff
@@ -156,7 +156,7 @@ func TestServer_TestIDProviderInstanceReduces(t *testing.T) {
 			UsernameMapping:    idp.OIDCMappingField_OIDC_MAPPING_FIELD_EMAIL,
 			AutoRegister:       true,
 		})
-		after := time.Now().Add(time.Second * 30) // need to allow time for the events to be processed
+		after := time.Now().Add(time.Second * 30)
```

Comment:
Removing the comment does not change the fact that the updated at column could be wrong. The database uses the NOW() function instead of the creation date of the event in these cases.

To fix this issue you need to:
1. do the update
2. read the changed rows
3. check if the updated at is correct
4. do another update of the row(s) if the updated at is not the creation date of the event